assignments/lambda-classes.js

```diff
@@ -1 +1,182 @@
+/* eslint-disable no-undef */
+/* eslint-disable no-console */
 // CODE here for your Lambda Classes
+
+//Classes
+class Person {
+  constructor(attributes) {
+    this.age = attributes.age;
+    this.name = attributes.name;
+    this.location = attributes.location;
+  }
+
+  speak() {
+    console.log(`Hello my name is ${this.name}, I am from ${this.location}`);
+  }
+}
+
+class Instructor extends Person {
+  constructor(attributes) {
+    super(attributes);
+    this.specialty = attributes.specialty;
+    this.favLanguage = attributes.favLanguage;
+    this.catchPhrase = attributes.catchPhrase;
+  }
+
+  demo(str) {
+    console.log(`Today we are learning about`, str);
+  }
+
+  grade(student, subject) {
+    console.log(`${student.name} receives a perfect score on ${subject}`);
+  }
+
+  //Stretch
+  randomlyGrade(student) {
+    const oldGrade = student.grade;
+    student.grade += Math.floor(Math.random() * 20 - 10);
+    console.log(
+      `${student.name}'s grade went from ${oldGrade} to ${student.grade}`
+    );
+  }
+}
+
+class Student extends Person {
+  constructor(attributes) {
+    super(attributes);
+    this.previousBackground = attributes.previousBackground;
+    this.className = attributes.className;
+    this.favSubjects = attributes.favSubjects;
+    //Stretch
+    this.grade = attributes.grade;
+  }
+
+  listsSubjects() {
+    this.favSubjects.forEach(element => console.log(element));
+  }
+
+  PRAssignment(subject) {
+    console.log(`${this.name} has submitted a PR for ${subject}`);
+  }
+
+  sprintChallenge(subject) {
+    console.log(`${this.name} has begun a spring challenge for ${subject}`);
+  }
+
+  //Stretch
+  graduate() {
+    if (this.grade > 70) {
+      console.log(`Congratulations ${this.name}! You've graduated!`);
+    } else {
+      console.log(`Whoops! You're not ready to graduate yet! Back to work!`);
+    }
+  }
+}
+
+class ProjectManager extends Instructor {
```

Comment:
Great job using a consistent naming convention across all of the constructors/supers!